client/src/components/Questions/qna.css

```diff
@@ -4,7 +4,7 @@ strong {
 #qna {
```

Comment:
We'll probably need to have a discussion as a group as to which font family we want to use and then just put that in the main CSS file, but that's for future improvements / refactoring.